conbench/entities/summary.py

```diff
@@ -19,12 +17,12 @@
 from ..entities.case import Case
 from ..entities.commit import Commit, get_github_commit, repository_to_url
 from ..entities.context import Context
-from ..entities.data import Data
+from ..entities.data import Data  # noqa: F401
```

Comment:
TIL about F401 and being able to disable _specific_ qas

Reply:
These lines will go away in the next PR when I drop these tables. I need them here because of tests and how drop_all() works.

Reply:
These lines will go away in the next PR when I drop these tables. I need them here because of tests and how drop_all() works.